components/Button.js

```diff
@@ -1,39 +1,62 @@
 import React from 'react'
+
 import { COLORS } from '../lib/constants'
 
-export default props => (
-  <button
-    onClick={props.onClick}
-    style={{
-      ...props.style,
-      background: COLORS.BLACK,
-      color: props.color,
-      boxShadow: `inset 0px 0px 0px ${props.selected ? 2 : 1}px ${props.color}`,
-      border: 'none'
-    }}
-    disabled={props.disabled}
-  >
-    <span>{props.title}</span>
+const Button = ({
+  onClick = () => {},
+  className = '',
+  background = COLORS.BLACK,
+  color = COLORS.SECONDARY,
+  hoverBackground = COLORS.HOVER,
+  hoverColor,
+  disabled,
+  notAllowed,
+  selected,
+  title,
+  children,
+  border,
+  center,
+  large,
+  style = {},
+  flex = 1,
+  padding = 0,
+  margin = 0
+}) => (
+  <button onClick={onClick} className={className} disabled={disabled || notAllowed} style={style}>
+    {title && <span className="button-title">{title}</span>}
```

Comment:
What is the difference between title and children? It seems like this shouldn't be the job of the button component, but rather it should just take the children passed in and render them directly. Do you think having both is a better API?

Reply:
Yeah not needed. I kind of like not having to use a closing tag if it's just text as children, but that's the only benefit it's providing right now.

Reply:
I would rather simplify the Button API as much as possible